main.js

```diff
@@ -176,7 +176,7 @@
       const transactions = this.props.data.transactions.filter(transaction => transaction.deleted);
       const result = [];
       for (var i = 0; i < transactions.length; i++) {
-        const transaction = transactions[i];
+        var transaction = transactions[i];
```

Comment:
Nit: could we keep transaction const and create a mutable memo variable like we did with payeeName?